ambuda/translations/sa/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,183 @@
+msgid "Ambuda"
+msgstr "अम्बुदः"
+
+#: ambuda/templates/include/footer.html:24
+#: ambuda/templates/include/header-login.html:20
+#: ambuda/templates/include/header.html:22 ambuda/templates/index.html:44
+#: ambuda/templates/texts/index.html:29
+msgid "Texts"
+msgstr "ग्रन्थाः"
+
+#: ambuda/templates/include/footer.html:25
+#: ambuda/templates/include/header-login.html:21
+#: ambuda/templates/include/header.html:23 ambuda/templates/index.html:45
+msgid "Dictionaries"
+msgstr "शब्दकोशाः"
+
+#: ambuda/templates/include/footer.html:26
+#: ambuda/templates/include/header-login.html:22
+#: ambuda/templates/include/header.html:24 ambuda/templates/index.html:46
+msgid "Proofing"
+msgstr "पाठशोधनम्"
+
+#: ambuda/templates/about/index.html:15 ambuda/templates/include/footer.html:34
+#: ambuda/templates/include/header-login.html:23
+#: ambuda/templates/include/header.html:25 ambuda/templates/index.html:47
+msgid "About"
+msgstr "अध्यम्बुदम्"
+
+#: ambuda/templates/about/privacy.html:10
+#: ambuda/templates/include/footer.html:42
+msgid "Privacy Policy"
+msgstr "गोपनीयतानीतिः"
+
+#: ambuda/templates/about/terms.html:9 ambuda/templates/include/footer.html:41
+msgid "Terms of Use"
+msgstr "उपयोगनियमाः"
+
+#: ambuda/templates/include/footer.html:27
+msgid "Support"
+msgstr "सहकारः"
+
+#: ambuda/templates/include/footer.html:28
+msgid "Donate"
+msgstr "दानम्"
+
+#: ambuda/templates/include/footer.html:29
+msgid "Contact"
+msgstr "सम्पर्कः"
+
+#: ambuda/templates/include/footer.html:36
+msgid "Mission"
+msgstr "लक्ष्याणि"
+
+#: ambuda/templates/include/footer.html:37
+msgid "Values"
+msgstr "स्थूलध्येयानि"
+
+#: ambuda/templates/include/footer.html:38
+msgid "Roadmap"
+msgstr "कार्ययोजना"
+
+#: ambuda/templates/include/footer.html:39
+msgid "People"
+msgstr "अग्रण्यः"
+
+#: ambuda/templates/include/footer.html:40
+msgid "Code and Data"
+msgstr "पाठ-तन्त्र-मूलानि"
+
+#: ambuda/templates/include/footer.html:47
+msgid "Community"
+msgstr "समुदायः"
+
+#: ambuda/templates/include/header-login.html:28
+msgid "Sign out"
+msgstr "निर्गमः"
+
+#: ambuda/templates/include/header-login.html:32
+msgid "Register"
+msgstr "पञ्जीकरणम्"
+
+#: ambuda/templates/include/header-login.html:33
+msgid "Sign in"
+msgstr "प्रवेशः"
+
+#: ambuda/templates/macros/library.html:5
+msgid "Contents"
+msgstr "अनुक्रमणिका"
+
+#: ambuda/templates/macros/library.html:6
+msgid "Resources"
+msgstr "पाठमूलानि"
+
+#: ambuda/templates/macros/proofing.html:35
+msgid "Help"
+msgstr "शोधसाहाय्यम्"
+
+#: ambuda/templates/macros/proofing.html:37
+msgid "Beginner's guide"
+msgstr "आरम्भसूचनाः"
+
+#: ambuda/templates/macros/proofing.html:38
+msgid "Complete guide"
+msgstr "पूर्णसूचनाः"
+
+#: ambuda/templates/macros/proofing.html:39
+msgid "Editor manual"
+msgstr "शोधकानुशाशनम्"
+
+#: ambuda/templates/macros/proofing.html:42
+msgid "Contribute"
+msgstr "योगदानम्"
+
+#: ambuda/templates/macros/proofing.html:44
+msgid "Ongoing projects"
+msgstr "सक्रियपरियोजनाः"
```

Comment:
परियोजनाः → प्रकल्पाः? (एवम् अन्यत्र)

Reply:
कुत्र? एवम् अस्ति संस्कृतभारत्याम् https://www.samskritabharati.in/project

Reply:
आः, विकिस्रोतसि विद्यते